src/testcases/common_localcosmosnet.ts

```diff
@@ -53,6 +60,18 @@ const walletSet = async (
     prefix,
   ),
 });
+const walletSetQa = async (
+  sdk: cosmosclient.CosmosSDK,
+  prefix: string,
+  mnemonicQA: string,
+): Promise<Record<string, Wallet>> => ({
+  genQaWal1: await mnemonicToWallet(
+    cosmosclient.AccAddress,
+    sdk,
+    mnemonicQA,
+    prefix,
+  ),
+});
```

Comment:
You can simplify this function
```suggestion
const walletSetQa = async (
  sdk: cosmosclient.CosmosSDK,
  prefix: string,
  mnemonicQA: string,
): Promise<Record<string, Wallet>> => await mnemonicToWallet(
    cosmosclient.AccAddress,
    sdk,
    mnemonicQA,
    prefix,
  );
```